api/src/main/java/org/openmrs/module/pihcore/setup/LocationTagSetup.java

```diff
@@ -109,6 +109,8 @@ private static void setupLocationTagsForLiberia(LocationService locationService,
 
         Field locationField = LiberiaLocations.class.getField(config.getSite().name().toString());
         LocationDescriptor location = (LocationDescriptor) locationField.get(LocationDescriptor.class);
```

Comment:
So I believe what this does is fetch the name of the current server site (either Pleebo or Harper) and then pulls that location descriptor from the LiberiaLocations class.  So on the Pleebo server, all these tags get added to Pleebo, and on the Harper server, all these tags get added to Harper.

With what you've added, the ED location tags will get added to the "harper" location on both the Pleebo and Harper servers, I believe... I don't think that's a big deal (I think the main concern is we don't want Harper to be tagged as a "Login location" on Pleebo or vice versa, or both will appear on the Login screen as options)  but if we can maintain the existing pattern it's probably better for consistency? Or we could potentially change it up so that we enable all the tags on for both locations on both servers, except for login?

 (If the ED component isn't enabled on Pleebo it still won't show up, even if the tags are there, so we don't need to worry about ED mistakenly being enabled on Pleebo.)

Reply:
Thanks for the correction. I have made the correction